consensus/src/transaction_filter/mod.rs

```diff
@@ -97,7 +97,9 @@ mod test {
 
     fn get_module_address(txn: &SignedTransaction) -> AccountAddress {
         match txn.payload().executable_ref() {
-            Ok(TransactionExecutableRef::EntryFunction(entry_func)) => {
+            Ok(TransactionExecutableRef::EntryFunction(entry_func))
```

Comment:
nit: it's probably fine to leave these changes as-is. I'm working through the file anyway 😄